app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
+  validates :text, presence: true
+  belongs_to :user
+  belongs_to :post
+
+  private
+
+  def update_counter_for_comments
```

Comment:
- [ ] Great effort working on your project, kindly give the `update_counter_for_comment` the after_save method just the way you gave the likes and posts model